src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/unit/service/ReferralServiceTest.kt

```diff
@@ -123,6 +124,9 @@ class ReferralServiceTest {
   @MockK(relaxed = true)
   private lateinit var organisationRepository: OrganisationRepository
 
+  @MockK(relaxed = true)
+  private lateinit var referralViewRepository: ReferralViewRepository
```

Comment:
Should tests be updated to use this repo?

Reply:
I'll update these in part 2 - Have done integration testing but want John to have a look before I tidy up all the existing code etc.